test/grape_test.rb

```diff
@@ -0,0 +1,85 @@
+require 'test_helper'
+require 'grape'
+require 'grape/active_model_serializers'
+
+class ActiveModelSerializers::GrapeTest < Minitest::Test
+  include Rack::Test::Methods
+
+  class GrapeTest < Grape::API
+    format :json
+    include Grape::ActiveModelSerializers
+
+    resources :grape do
+      get '/render' do
+        render ARModels::Post.new(title: 'Dummy Title', body: 'Lorem Ipsum')
+      end
+
+      get '/render_with_json_api' do
+        post = ARModels::Post.new(title: 'Dummy Title', body: 'Lorem Ipsum')
+        render(post, meta: { page: 1, total_pages: 2 }, adapter: :json_api)
+      end
+
+      get '/render_array_with_json_api' do
+        post = ARModels::Post.create(title: 'Dummy Title', body: 'Lorem Ipsum')
+        post.dup.save
+        render(ARModels::Post.all, adapter: :json_api)
+      end
+    end
+  end
+
+  def app
+    GrapeTest.new
+  end
+
+  def test_formatter_returns_json
+    get '/grape/render'
+    assert last_response.ok?
+    expected = { id: nil, title: 'Dummy Title', body: 'Lorem Ipsum', comments: [], author: nil }
+    assert_equal expected.to_json, last_response.body
+  end
+
+  def test_render_helper_passes_through_options_correctly
+    get '/grape/render_with_json_api'
+
+    post = ARModels::Post.new(title: 'Dummy Title', body: 'Lorem Ipsum')
+    serializable_resource = serializable(post, serializer: ARModels::PostSerializer, adapter: :json_api, meta: { page: 1, total_pages: 2 })
```

Comment:
I'd just like to note that the other two tests here have an `expected` hash but this one generates the expected hash.   I'm not sure if I like one style over the other or if there's a reason they're different. So, just noting it in case you have any thoughts.